src/m4st/callhome/process_mappings.py

```diff
@@ -31,12 +31,25 @@ def reconcile_timestamps(timestamps: list[str]):
     return "_".join([str(min_min), str(max_max)])
 
 
-def process(cha_root_dir, mapping_file, eng_file) -> pd.DataFrame:
+def process(cha_root_dir: str, mapping_file: str, eng_file: str) -> pd.DataFrame:
     r"""
     Loads a DataFrame of Spanish utterances alongside their English translations.
 
     This function expects that the dataset has been pre-processed, so that the
     lines in the mapping file match to the correct lines in the transcription files.
+
+    Args:
+        cha_root_dir (str): Path to folder containing the .cha transcript files.
```

Comment:
Minor but generally if you've type-hinted the args in the function def then it's not necessary to repeat the type in the docstring (and makes things easier to maintain).